Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ocaml/opam
+
+WORKDIR learn-ocaml
+
+ADD . .
+
+RUN sudo chown -R opam:nogroup .
+
+RUN cd /home/opam/opam-repository && git pull
+
+RUN opam config exec -- opam update
+
+RUN opam config exec -- make build-deps
+
+RUN opam config exec -- make REPO_DIR=exercises_repository
+
+EXPOSE 9090
+
+CMD /home/opam/learn-ocaml/learnocaml-simple-server.byte -port 9090
```

Comment:
According to https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact shouldn't this line be replaced with this?
```
ENTRYPOINT ["/home/opam/learn-ocaml/learnocaml-simple-server.byte", "-port", "9090"]
CMD []
```

Reply:
Or if the `opam config exec --` prefix is needed and you'd like to rely on [this ENTRYPOINT](https://github.com/ocaml/opam-dockerfiles/blob/5fd5de2854fd944757d3d84ef1fce2604f71631c/Dockerfile#L26), maybe the best choice would be
```
CMD ["/home/opam/learn-ocaml/learnocaml-simple-server.byte", "-port", "9090"]
```